### PR TITLE
[Addressing] Province rendering tweaks

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | #3902
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/AddressingBundle/Form/Type/ProvinceType.php [hunk 1]
```diff
@@ -30,6 +30,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('name', 'text', array(
                 'label' => 'sylius.form.province.name',
             ))
+            ->add('abbreviation', 'text', array(
+                'label' => 'sylius.form.province.abbreviation',
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/AddressingBundle/Form/Type/ProvinceType.php]: It's not required so we should set it to false here.

**Comment by pjedrzejewski** [comment]
Looks good to me, let's fix the build, merge it and add caching in another PR.

File: src/Sylius/Bundle/AddressingBundle/spec/Form/EventListener/BuildAddressFormSubscriberSpec.php [hunk 2]
```diff
@@ -64,20 +64,22 @@ function it_adds_or_removes_provinces_on_pre_set_data(
...
         $province->getCode()->willReturn('province');
-        $address->getProvince()->willReturn('province');
+        $address->getProvinceCode()->willReturn('province');
 
-        $factory->createNamed('province', 'sylius_province_code_choice', 'province', Argument::withKey('country'))
+        $formFactory->createNamed('provinceCode', 'sylius_province_code_choice', 'province', Argument::withKey('country'))
             ->willReturn($provinceForm);
 
+
+
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AddressingBundle/spec/Form/EventListener/BuildAddressFormSubscriberSpec.php]: Redundant blank spaces

**Comment by michalmarcinkowski** [comment]
Add fallback to `code` in `ProvinceNamingProvider::getAbbreviation` and good to go :)

**Comment by peteward** [comment]
Thanks for getting this done so quickly :) :+1:

**Comment by pjedrzejewski** [comment]
Thank you Jasiek! :+1: :)

**Pull request merged**